client/MonitoringServiceApiCaller/AutoRegistrationInMonitoring.cs

```diff
@@ -93,5 +97,90 @@ public static class AutoRegistrationInMonitoring
                 log.WriteMonitor("Auto-registration in monitoring", podTag, ex.ToAsyncString());
             }
         }
+
+        /// <summary>
+        /// Registers calling application in monitoring service based on application url from environemnt variable.
+        /// </summary>
+        /// <param name="configuration">Application configuration that is used for environemnt variable search.</param>
+        /// <param name="monitoringServiceUrl">Monitoring service url.</param>
+        /// <param name="healthNotifier">Health notifier</param>
+        /// <returns></returns>
+        public static async Task RegisterAsync(
```

Comment:
Maybe it is better make it an extension method?

Reply:
If we make it extension methid then we need to rename it as some thing like - RegisterInMonitoringAsync